ajax/libs/gibon/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "gibon",
+  "version": "0.3.2",
```

Comment:
If we add library via single package.json, we should remove version field. Thanks.